index.js

```diff
@@ -5,6 +5,18 @@ function fixCapitalisedPlural(fn) {
   };
 }
 
+function fixChangePlural(fn) {
+  return function(str) {
+    const matches = str.match(/[A-Z_][a-z0-9]*$/);
```

Comment:
This can currently result in an empty string being fed into pluralize (if the string ends with a `_` 🙄 ); let's be a bit stricter:
```suggestion
    const matches = str.match(/(?:[A-Z][a-z0-9]*|_[a-z0-9]+)$/);
```